SocketRocket/SRWebSocket.m

```diff
@@ -671,7 +678,7 @@ - (void)_closeWithProtocolError:(NSString *)message;
     // Need to shunt this on the _callbackQueue first to see if they received any messages 
     [self _performDelegateBlock:^{
         [self closeWithCode:SRStatusCodeProtocolError reason:message];
-        dispatch_async(_workQueue, ^{
+        dispatch_async(self->_workQueue, ^{
```

Comment:
again, the `self->` changes do not seem like they are necessary. Please remove them from your PR.